interceptor.go

```diff
@@ -25,25 +25,31 @@ import (
 // UnaryInvoker is called by UnaryClientInterceptor to complete RPCs.
 type UnaryInvoker func(ctx context.Context, method string, req, reply interface{}, cc *ClientConn, opts ...CallOption) error
 
-// UnaryClientInterceptor intercepts the execution of a unary RPC on the client. invoker is the handler to complete the RPC
-// and it is the responsibility of the interceptor to call it.
+// UnaryClientInterceptor intercepts the execution of a unary RPC on the client. The interceptors can be set on the
+// client connection using WithUnaryInterceptor or WithChainUnaryInterceptor. If client connections has the interceptor set,
+// then it delegates all unary RPC invocations to the interceptor, and it is the responsibility of the interceptor to call
+// invoke.
 //
-// Experimental
+// method is RPC name. req and reply are corresponding request and response RPC messages. cc is the client connection
```

Comment:
Nit: Could you also please make sure the comment lines are wrapped at 80 cols. Thanks.